mbf_abstract_nav/src/controller_action.cpp

```diff
@@ -203,14 +206,7 @@ void ControllerAction::run(GoalHandle &goal_handle, AbstractControllerExecution
       case AbstractControllerExecution::PLANNING:
         if (execution.isPatienceExceeded())
         {
-          ROS_INFO_STREAM_NAMED(name_, "The controller patience has been exceeded! Stopping controller...");
-          // TODO planner is stuck, but we don't have currently any way to cancel it!
-          // We will try to stop the thread, but does nothing with DWA, TR or TEB controllers
-          // Note that this is not the same situation as in case AbstractControllerExecution::PAT_EXCEEDED,
```

Comment:
This part of the comment is still valuable, imho, as it can be confusing the two different cases of patience exceeded